src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.WriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WriterServiceImpl implements WriterService {
+    private final String fileName = "src/main/resources/report.csv";
+
+    @Override
+    public void writeToFile(List<Fruit> fruits) {
```

Comment:
also split this logic into two services: generating report to string and writing this string to file